scripts/skills/perks/perk_rf_combo.nut

```diff
@@ -0,0 +1,81 @@
+this.perk_rf_combo <- ::inherit("scripts/skills/skill", {
+	m = {
+		UsedSkillID = "",
+		UsedSkillName = ""
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_combo";
+		this.m.Name = ::Const.Strings.PerkName.RF_Combo;
+		this.m.Description = ::Const.Strings.PerkDescription.RF_Combo;
```

Comment:
The perk description is not suited to be the header description of this perk in effect form when viewing the tooltip